app/views/groups/show.html.erb

```diff
@@ -0,0 +1,39 @@
+<div class="transactions-container">
+  <div class="transactions-header">
+    <a href="/"><i class="fas fa-arrow-left fa-2x"></i></a>
+    <h2>TRANSACTIONS</h2>
+  </div>
+
+  <% if notice %>
+    <p style="color: green"><%= notice %></p>
+  <% end %>
+
+  <div class="transactions-inner-container">
+    <div class="group-container">
+      <div>
+        <%=image_tag "#{@group.icon}"%>
+        <p><%= @group.name %></p>
+      </div>
+      <% total_amount = 0.00 %>
+      <% @group.payment_groups.each do |element| %>
+        <% total_amount += element.payment.amount %>
+      <% end %>
+      <p>Total: $<%= total_amount %></p>
+      <%= button_to "Delete", group_path(@group), :method => :delete %>
+    </div>
+
+    <h3>Transactions</h3>
+    
+    <div class="payments-container">
+      <% @payments_group.each do |element|%>
+        <div class="payment">
+          <p><%= element.payment.name %></p>
+          <p>$<%= element.payment.amount %></p>
+          <%= button_to "Delete", payment_path(element.payment), :method => :delete %>
+        </div>
+      <% end %>
+    </div>
+  </div>
+
```

Comment:
- [ ] Please make sure to stick with the design. Include date when transaction created and also keep font small while viewing on a small screen.

### Your design
![Screenshot from 2022-03-16 21-52-32](https://user-images.githubusercontent.com/22434660/158640439-e5d5c626-6f99-4f29-bee1-a1ebeb4b24e4.png)

### Given design

![Screenshot from 2022-03-16 21-52-14](https://user-images.githubusercontent.com/22434660/158640406-85c2b5f4-64f8-4d24-9546-639a71f7224c.png)

Reply:
Ok thank you will do that.